### PR TITLE
Add cloud-init foo to install GoPhish TLS certs

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds some `cloud-init` foo to install the GoPhish TLS certificates.

## 💭 Motivation and context ##

These changes are required for cisagov/cool-system-internal#81.

See also:
- cisagov/pca-gophish-composition-packer#72

## 🧪 Testing ##

All automated tests pass.  I also deployed a new staging AMI with these changes, deployed it to `env6-staging`, and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: gophish_cloud_init.tf [hunk 1]
```diff
@@ -132,6 +132,30 @@ data "cloudinit_config" "gophish_cloud_init_tasks" {
     merge_type   = "list(append)+dict(recurse_array)+str()"
   }
 
+  # Install certificate for postfix.
```
Comment (dav3r) [inline gophish_cloud_init.tf]:
```suggestion
  # Install certificate for Gophish.
```

File: gophish_cloud_init.tf [hunk 2]
```diff
@@ -132,6 +132,30 @@ data "cloudinit_config" "gophish_cloud_init_tasks" {
     merge_type   = "list(append)+dict(recurse_array)+str()"
   }
 
+  # Install certificate for Gophish.
+  part {
+    content = templatefile(
+      "${path.module}/cloud-init/install-certificates.py", {
```
Comment (mcdonnnj) [inline gophish_cloud_init.tf]: So small nitpick, but with this change in usage should this file be renamed to `install-certificates.tpl.py`?

**Review by mcdonnnj — APPROVED** [review]
Seems reasonable enough. I did have one question below.

File: gophish_cloud_init.tf (hunk 2 above)
Comment (jsf9k) [reply gophish_cloud_init.tf]: Please see commit 7a5f278.

**Review by dav3r — APPROVED** [review]
Looks great to me!  🦀

**Pull request merged**